pom.xml

```diff
@@ -20,11 +20,22 @@
   <name>GitHub API Plugin</name>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/GitHub+API+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.615</jenkins.version>
```

Comment:
Well, most users of earlier lines use Java 7 or 8 anyway. OTOH pending [JENKINS-20679](https://issues.jenkins-ci.org/browse/JENKINS-20679) the only way to prevent accidental usage on Java 6 is to update the Jenkins version. So +0 on this.

Anyway, if you are going to update it, may as well go to the next LTS line: 1.625.x.